airflow/contrib/hooks/azure_container_instance_hook.py

```diff
@@ -92,43 +92,28 @@ def create_or_update(self, resource_group, name, container_group):
                                                           name,
                                                           container_group)
 
+    from zope.deprecation import deprecation
```

Comment:
Move to top of file please.

Reply:
I noticed it was placed further down here:
https://airflow.apache.org/_modules/airflow/sensors.html

I figured it was placed there so it would be easier to tidy up along with the function in the next update. Should I still move the import?

Reply:
Yes please - that is a special case (to not import it unless that setting is enabled), in other places we use it like a normal import, and the Flake8 check will tell us about unused imports once we remove the deprecation.